fullstack-cert/typescript-projects/resume-builder/src/App.tsx

```diff
@@ -376,20 +340,14 @@ export const EducationForm: React.FC<EducationFormProps> = ({
     <div className="bg-white rounded-lg shadow-sm border border-slate-200 p-6">
       <div className="flex items-center justify-between mb-6">
         <h2 className="text-xl font-semibold text-slate-800 flex items-center">
-          <img
-            src="https://unpkg.com/lucide-static@0.541.0/icons/graduation-cap.svg"
-            className="w-5 h-5 mr-2 text-blue-600 blue-filter"
-          />
+          <i className="fa-solid fa-graduation-cap w-5 h-5 mr-2 text-[16px] text-[#2563EF]"></i>
```

Comment:
better to add `w12 h12 mx-auto mb-4 text-[40px]` class here to make this icon larger

Reply:
This is the blue cap in the header area, NOT the grey cap that appears when there is "No education added yet."

Reply:
Looks like the actual issue is that the size for it says `font` instead of `text`. I must have caught it with a find and replace towards the end of the night and not noticed it hit the wrong element. That's on line 489.